mentoring/templates/html/mentoring_grade.html

```diff
@@ -0,0 +1,16 @@
+<script type="text/template" id="xblock-grade-template">
+  <% if (_.isNumber(max_attempts) && max_attempts > 0 && num_attempts >= max_attempts) {{ %>
+  <p>Note: you have used all attempts. Continue to the next unit.</p>
+  <% }} else {{ %>
+  <p>Note: if you retake this assessment, only your final score counts.</p>
+  <% }} %>
+  <h2>You scored <%= score %>% on this assessment.</h2>
+  <hr/>
+
+  <span class="assessment-checkmark icon-2x checkmark-correct icon-ok fa fa-check"></span>
+  <p>You answered <%= correct_answer %> questions correctly.</p>
+  <span class="assessment-checkmark icon-2x checkmark-partially-correct icon-ok fa fa-check"></span>
+  <p>You answered <%= partially_correct_answer %> questions partially correct.</p>
+  <span class="assessment-checkmark icon-2x checkmark-incorrect icon-exclamation fa fa-exclamation"></span>
+  <p>You answered <%= incorrect_answer %> questions incorrectly.</p>
```

Comment:
@bradenmacdonald Product did an initial review based on the screenshots - it went well, besides the theming changes that are underway, there was only one issue related to the features, which is to fix the pluralization here when there is only one correct/incorrect answer.